<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGameSettings</class>
 <widget class="QDialog" name="NewGameSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 255);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGameMode">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPVP">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>P V P</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPVE">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>P V E</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelColor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelBlack">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>BLACK</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelRandom">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>RANDOM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelWhite">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>WHITE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelLevel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Level</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEasy">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>EASY</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelMedium">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>MEDIUM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHard">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>HARD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Image">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewGameSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewGameSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
